platforms/index.rst

```diff
@@ -6,6 +6,21 @@ Supported Platforms
 Platform and board specific support is continually added to SOF project as
 documented below.
 
+.. csv-table:: Supported Platforms (in order of upstreaming)
+   :header: "Platform", "Architecture", "Cores/Clocks", "Memory", "Audio Interfaces"
+   :widths: 20, 20, 10, 10, 20
+
+   "Intel Baytrail / Merrifield", "Xtensa HiFi2 EP", "1 @ 50 - 400MHz", "96/192kB I/DRAM", "3*SSP (I2S, PCM)"
+   "Intel Cherrytrail / Braswell", "Xtensa HiFi2 EP", "1 @ 50 - 400MHz", "96/192kB I/DRAM", "6*SSP (I2S, PCM)"
+   "Host Testbench", "PC", "N/A", "N/A", "N/A Files are used to simulate DAIs"
+   "Intel Broadwell", "Xtensa HiFi2 EP", "1 @ 50 - 400MHz", "320/640kB I/DRAM", "2*SSP (I2S, PCM)"
+   "Intel Apollolake", "Xtensa HiFi3", "2 @ 50 - 400MHz", "128/1024kB LP/HP SRAM", "6*SSP (I2S, PCM), HDA, DMIC"
+   "Intel Cannonlake", "Xtensa HiFi3", "4 @ 50 - 400MHz", "128/1024kB LP/HP SRAM", "6*SSP (I2S, PCM), HDA, DMIC"
+   "Intel Suecreek", "Xtensa HiFi3", "2 @ 50 - 400MHz", "128/1024kB LP/HP SRAM", "6*SSP (I2S, PCM), DMIC"
+   "Intel Icelake", "Xtensa HiFi4", "4 @ 50 - 400MHz", "128/1024kB LP/HP SRAM", "6*SSP (I2S, PCM), HDA, DMIC"
+   "NXP i.MX8", "Xtensa HiFi3", "1 @ ???MHz", "??? SRAM", "EASI ??"
+   "Intel Tigerlake", "Xtensa HiFi4", "4 @ 50 - 400MHz", "128/1024kB LP/HP SRAM", "6*SSP (I2S, PCM), HDA, DMIC"
+
```

Comment:
-> NXP i.MX8/i.MX8X, "Xtensa HiFi4", 1 @ 666MHz, "64 KB TCM / 448 KB OCRAM / 8MB SDRAM", 1 x ESAI, 1 x SAI

Reply:
Thanks @dbaluta

Reply:
@lgirdwood one more thing. Maybe we can reorder all the entries so that we have Intel platforms in one place, then Host, then NXP.

Reply:
I can do that.